quiz.rb

```diff
@@ -13,25 +13,48 @@
 # Question 1
 # Take a string and return an array of normalized words.
 # normalized words contain no punctuation and are all lower case
+
+
 def normalize_words words
+  words = paragraph.split(/\W+/)
 end
 
 # Question 2
 # Take a string and return the count of normalized words
 def word_count words
+  count = 0
+  words.each do |words|
```

Comment:
nice enumerator attempt here, but just use `.length`. this would have worked if you ended you `.each` block with `end`